### PR TITLE
Leftover Sprites Fix

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR removes some leftover sprites from a previous PR.
there was apparently a unused duplicate sprite file, which is the one the previous PR removed from.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Comment by alsoandanswer** [comment]
please wait until my medbot pr gets merged

**Comment by SleepyGemmy** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**
